src/main/java/com/softserve/academy/spaced/repetition/controller/dto/annotations/PriceEntityInterface.java

```diff
@@ -0,0 +1,10 @@
+package com.softserve.academy.spaced.repetition.controller.dto.annotations;
+
+import com.softserve.academy.spaced.repetition.domain.BasePrice;
+
+public interface PriceEntityInterface {
+
+    BasePrice getPriceObject();
```

Comment:
This method:

- must be removed in the interface, as it is not accessed outside of implementation.

- can be removed in the implementation.